redstork/page.py

```diff
@@ -98,19 +98,14 @@ def render_(self, file_name, scale=1.0):
         if result == 0:
             raise RuntimeError('Failed to render as ' + file_name)
 
-    def render_to_buffer(self, scale=1.0, rect=None):
-        '''Render page (or rectangle on the page) as PPM image file.
-
-        Args:
-            scale (float):      scale to use (default is 1.0, which will assume that 1pt takes 1px)
-            rect (tuple):       optional rectangle to render. Value is a 4-tuple of (x0, y0, x1, y1) in PDF coordinates.
-                                if None, then page's :attr:`crop_box` will be used for rendering.
+    def get_fsmatrix_(self, rotation, crop_box, rect, scale):
```

Comment:
Please rename to `_get_fsmatrix`, as this is an internal methopd (we do not document it and users are not supposed to use it).

Reply:
Sorry for a naive conflict